bitnami/external-dns/templates/clusterrole.yaml

```
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/{{ .Values.rbac.apiVersion }}
 kind: ClusterRole
 metadata:
-  name: {{ template "external-dns.fullname" . }}
+  name: {{ printf "%s-%s" (include "common.names.fullname.namespace" .) "external-dns" }}
```

I think what you intended to do is:

```suggestion
  name: {{ template "common.names.fullname.namespace" . }}
```